api/models/Reference.js

```diff
@@ -40,7 +40,39 @@ module.exports = {
     approved: "boolean",
     edited: "boolean",
     notes: "string",
-    privatenotes: "string"
```

Comment:
It appears you have included the commit that I got rid of :-P please remove this and the other commit that you've stolen for some reason.